realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -662,6 +685,13 @@ public static String getTableName() {
                 ((AllTypesRealmProxyInterface) obj).realmSet$columnBinary(JsonUtils.stringToBytes(json.getString("columnBinary")));
             }
         }
+        if (json.has("columnCounter")) {
+            if (json.isNull("columnCounter")) {
+                throw new IllegalArgumentException("Trying to set non-nullable field 'columnCounter' to null.");
```

Comment:
Is this the semantics we want? By doing this, it looks impossible to initialize a counter to `null` from JSON?